test/torchaudio_unittest/prototype/hdemucs_test_impl.py

```diff
@@ -106,3 +116,88 @@ def test_decoder_output_shape_time(self):
 
         assert z.size() == (batch_size, chout, t * stride)
         assert y.size() == (batch_size, chin, t)
+
+
+@skipIfNoModule("demucs")
+class CompareHDemucsOriginal(TorchaudioTestCase):
+    """Test the process of importing the models from demucs.
+
+    Test methods in this test suite will check to
+    1. Assure correctness in factory functions, comparing with original hybrid demucs
+    """
+
+    def _get_original_model(self, sources: List[str], nfft: int, depth: int):
+        from demucs import hdemucs as original
+
+        original = original.HDemucs(sources, nfft=nfft, depth=depth)
+        return original
+
+    @parameterized.expand(
+        [
+            (["bass", "drums", "other", "vocals"],),
+            (["bass", "drums", "other"],),
+            (["bass", "vocals"],),
+            (["vocals"],),
+        ]
+    )
+    def test_import_recreate_low_model_test(self, sources):
+        sample_rate = 8000
+        nfft = 1024
+        depth = 5
+        duration = 10
```

Comment:
`duration`, `channels`, and `batch_size` are reused across tests -- what about setting these values as default params in `_get_inputs` and avoid redefining them in test cases

Reply:
Since you still need to define and use these variables in the shape tests, I think the better approach would still be letting `_get_inputs` take in these parameters, but with a default value.

```
def _get_inputs(
    sample_rate: int,
    device: torch.device,
    batch: int = 1,
    duration: int = 10,
    channels: int = 2
)
```

Then, for the shape tests, since you declare the variables in the test, you can pass them into `_get_inputs` for consistency. But for the `import_recreate_xxx_tests`, since they are not explicitly defined and used in the tests, we can leave those tests as is to use the default values.